spotbugsTestCases/src/java/CheckedJSpecifyNullableReturn.java

```diff
@@ -0,0 +1,15 @@
+import org.jspecify.annotations.Nullable;
+
+public class CheckedJSpecifyNullableReturn {
```

Comment:
As far as I can see, the positive and the negative code examples are quite short and really similar, it only differs in the `bar()` function. IMO the two examples could be in one class. Or if, you prefer to keep them separate, it would be a good idea to move them to the same package (e.g. the [nullnessAnnotations](https://github.com/spotbugs/spotbugs/tree/master/spotbugsTestCases/src/java/nullnessAnnotations) package contains a similar classes), so it's easy to look for them.

Reply:
This is fine but I have a process question.  Random approvers keep popping up and adding requests and blocking this.  How do I know what is the actual requirements here?

Reply:
All the main active maintainers responded on this.  Nothing from the norm here.  We typically allow everyone to comment and once no more comments it will get approved / merged.  Main thing is that you are getting action here which is a good thing.

Reply:
Moved.